modules/client/examples/index.ts

```diff
@@ -77,7 +77,7 @@ export const contextParams: ContextParams = {
   // They will switch depending on the network (production, development)
   graphqlNodes: [
     {
-      url: "https://subgraph.satsuma-prod.com/aragon/core-goerli/api",
+      url: "https://subgraph.satsuma-prod.com/aragon/osx-goerli/api",
```

Comment:
Should we also add the API_KEY in the url??

Reply:
Does matter that much. The API is public built into the frontend after building and everybody with devtools can see it